selftests/functional/test_basic.py

```diff
@@ -1013,6 +1015,8 @@ class PluginsXunitTest(AbsPluginsTest, unittest.TestCase):
 
     def setUp(self):
         self.tmpdir = tempfile.mkdtemp(prefix='avocado_' + __name__)
+        tests_dir = os.path.dirname(os.path.abspath(__file__))
+        self.junit_schema_path = os.path.join(tests_dir, 'junit-4.xsd')
```

Comment:
Shouldn't we move the schema file to `selftests/.data` and reuse on both unit and functional tests?